lib/ecto/datetime.ex

```diff
@@ -167,6 +167,9 @@ defmodule Ecto.Date do
 
   @doc """
   Returns an `Ecto.Date` in local time.
+
+  WARNING: Using the local time of the server will often lead to
+  intermittent bugs. Please use the `utc/0` function instead.
```

Comment:
Can we expand on why it leads to bugs? Maybe at least say that local time may go back and forth in time and therefore it is not always increasing, due to DST changes?